library/translation.inc.php

```diff
@@ -321,4 +321,21 @@ function mb_strpad($input, $length, $pad = ' ', $type = STR_PAD_RIGHT, $charset
 return $output;
 }
 
+
+/**
+ * Set theme for RTL language direction in globals setting.
+ * @return {void}
+ * @author Amiel <amielel@matrix.co.il>
+ */
+function setRtlCssTheme(){
+    
+    //validate that user id exist 
+    if(!isset($_SESSION['authUserID']))return;
+     
+    if(!function_exists('setUserSetting'))require_once dirname(__FILE__) . '/user.inc';
+    
+    //set rtl theme 
+    setUserSetting('global:css_header', 'style_sky_blue_rtl.css', null, false); 
+}
```

Comment:
Would it make more sense to create another globals setting akin to Administration->Local->Theme that was below this setting and called something like Theme for RTL Languages. Then can also set it per user. And could only allow to choose from the themes that support RTL.